src/chrome/content/rules/thedailywtf.com.xml

```diff
@@ -0,0 +1,26 @@
+<ruleset name="thedailywtf.com">
+	<!-- Failing:
+	<target host="forums.thedailywtf.com" />
+	<target host="img.thedailywtf.com" />
+	<target host="live.thedailywtf.com" />
+	<target host="omg.thedailywtf.com" />
+	<target host="cms.omg2.thedailywtf.com" />
+	-->
```

Comment:
Please write out the reasons why these targets fail.

Reply:
Done.